os/tools/mksamsungheader.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+############################################################################
+#
+# Copyright 2021 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+############################################################################
+import os
+import sys
+
+TOOL_DIR = '/root/tizenrt/os/tools/'
```

Comment:
`/root/tizenrt/` is not right for all the cases. That's why we use `test -d ${0%/*} && cd ${0%/*}; pwd\` in shell script to get current path.
Please do like shell script.